### ISSUE TITLE
Line numbers for validation input

### ISSUE THREAD

**Issue by barabo** [hapifhir/org.hl7.fhir.validator-wrapper#59]
After using the online validator for the first time recently, I thought it would be really useful if the `textarea` with the to-be-validated content would also display line numbers - especially since the validation errors reference line numbers in the input.

![image](https://user-images.githubusercontent.com/4342684/124785739-b7bab400-df0c-11eb-80c3-3a53e9a2013c.png)

Perhaps something like [CodeMirror](https://codemirror.net/6/) could be added without too much trouble.

**Comment by markiantorno** [comment]
I agree with this completely. It will be the next issue I take on. Thank you for the feedback.
